client/src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj

```diff
@@ -1508,7 +1512,7 @@
 			repositoryURL = "https://github.com/getsentry/sentry-cocoa";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 7.31.3;
+				minimumVersion = 8.26.0;
```

Comment:
Make sure you also update macOS. They should be using the same version of the dependencies, since they share code.